dev-requirements.txt

```diff
@@ -1,5 +1,6 @@
 pytest~=7.0
 twine~=3.1
+databricks-sdk~=0.1.8
```

Comment:
Is it really necessary? Can't we test it using just fsspec?

Reply:
I talked with you about that, and I prefer the setup of the test to not depend on the code of what we should test. If I override`fsspec.databricksfilesystem`, I would rather set up the test using independent modules.